packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -677,6 +681,7 @@ export const handleClientNotifications = async (client, client_store, ui_store,
 
     if (loginid !== LocalStore.get('active_loginid')) return {};
 
+    await BinarySocket.wait('residence_list');
```

Comment:
is there a reason why we call for `residence_list` here?

Reply:
because we need to make sure we have the residence to avoid UI flash for popup and notification

Reply:
```suggestion
    await BinarySocket.wait('get_account_status');
```
I think account_status would be a more appropriate call to get the account details?